src/go/rdctl/cmd/set.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/spf13/cobra"
+	"os"
+	"strings"
+)
+
+type serverSettings struct {
+	Kubernetes struct {
+		ContainerEngine string `json:"containerEngine"`
+		Enabled         bool   `json:"enabled"`
+		Version         string `json:"version"`
+	} `json:"kubernetes,omitempty"`
+}
+
+var specifiedSettings serverSettings
+
+// setCmd represents the set command
+var setCmd = &cobra.Command{
+	Use:   "set",
+	Short: "Update selected fields in the Rancher Desktop UI and restart the backend.",
+	Long: `The following options are supported:
+    --container-engine=containerd|moby
+    --kubernetes-enabled=true|false
+    --kubernetes-version=VERSION
+
+The '=' sign can be replaced by one or more spaces.
```

Comment:
That's just standard cobra behaviour; we shouldn't document it here.
```
$P$G> .\resources\win32\bin\rdctl.exe set --help
The following options are supported:
    --container-engine=containerd|moby
    --kubernetes-enabled=true|false
    --kubernetes-version=VERSION

The '=' sign can be replaced by one or more spaces.
'docker' is an accepted synonym for 'moby'.

Usage:
  rdctl set [flags]

Flags:
      --container-engine string     Set engine to containerd or moby (aka docker).
  -h, --help                        help for set
      --kubernetes-enabled          Control whether kubernetes runs in the backend.
      --kubernetes-version string   Choose which version of kubernetes to run.

Global Flags:
      --config-path string   config file (default C:\Users\Mark\AppData\Roaming\rancher-desktop\rd-engine.json)
      --host string          default is localhost; most useful for WSL
      --password string      overrides the password setting in the config file
      --port string          overrides the port setting in the config file
      --user string          overrides the user setting in the config file
```

Reply:
Also gone. But I was surprised to find out that `--no-kubernetes-enabled` isn't an automatic addition. I've seen other libraries do that.